tests/frame/getitem_dunder_test.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pandas as pd
+import pytest
+
+import narwhals as nw
+
+
+class TestGetItemWorksForTupleIndexing:
+    """Tests added to address issue #990 - https://github.com/narwhals-dev/narwhals/issues/990"""
+
+    @pytest.fixture
+    def data(self) -> dict[str, list[int]]:
+        return {"x": [1, 2, 3]}
+
+    @pytest.fixture
+    def pd_df(self, data: dict[str, list[int]]) -> nw.DataFrame[Any]:
+        # TODO(mikeweltevrede): Use constructor with eager
```

Comment:
Still to do: use the constructor with eager mode

Reply:
Done!